tests/unit/config_tests.py

```diff
@@ -20,6 +20,8 @@ class TestConfigParser(unittest.TestCase):
         ('--keep-builddir', 'keep_builddir', True),
         ('--build-versions-repository-url=foo', 'build_versions_repository_url', 'foo'),
         ('--build-version=foo', 'build_version', 'foo'),
+        ('--mock-args=--enable-plugin=tmpfs --plugin-option=tmpfs:keep_mounted=True', 'mock_args',
```

Comment:
@olavphibm Wow that is some weird behavior there. We shouldn't be using 'nargs', then. Let's just take a string. From what I gathered, our usage of argparse is returning `['--enable-plugin=... --plugin-option=...']` and we later do `" ".join(mock_args)`, which works, but is misleading.

(We should still use '--foo --bar' for the tests instead of the real options, though.)